### PR TITLE
contrib/boosted_trees: minor spelling tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Review by frankchn — APPROVED** [review]

File: tensorflow/contrib/boosted_trees/lib/utils/batch_features.cc [hunk 1]
```diff
@@ -54,7 +54,7 @@ Status BatchFeatures::Initialize(
     TF_CHECK_AND_RETURN_IF_ERROR(
         dense_float_feature.dim_size(1) == 1,
         errors::InvalidArgument(
-            "Dense float features may not be multi-valent: dim_size(1) = ",
+            "Dense float features may not be multivalent: dim_size(1) = ",
```
Comment (brettkoonce) [inline tensorflow/contrib/boosted_trees/lib/utils/batch_features.cc]: test fixed here

File: tensorflow/contrib/boosted_trees/lib/utils/batch_features_test.cc [hunk 2]
```diff
@@ -59,7 +59,7 @@ TEST_F(BatchFeaturesTest, DenseFloatFeatures_Multivalent) {
   BatchFeatures batch_features(1);
   auto dense_vec = AsTensor<float>({3.0f, 7.0f}, {1, 2});
   auto expected_error = InvalidArgument(
-      "Dense float features may not be multi-valent: dim_size(1) = 2");
+      "Dense float features may not be multivalent: dim_size(1) = 2");
```
Comment (brettkoonce) [inline tensorflow/contrib/boosted_trees/lib/utils/batch_features_test.cc]: test fixed here

**Comment by frankchn** [comment]
tensorflow/contrib/boosted_trees/python/kernel_tests/prediction_ops_test.py:123: [C0301(line-too-long), ] Line too long (81/80)

File: tensorflow/contrib/boosted_trees/python/kernel_tests/prediction_ops_test.py [hunk 3]
```diff
@@ -120,8 +120,8 @@ def setUp(self):
     """Sets up the prediction tests.
 
     Create a batch of two examples having one dense float, two sparse float
-    single valued, one sparse float multidimensionl and one sparse int features.
-    The data looks like the following:
+    single valued, one sparse float multidimensional and one sparse int
+    features.  The data looks like the following:
```
Comment (brettkoonce) [inline tensorflow/contrib/boosted_trees/python/kernel_tests/prediction_ops_test.py]: @frankchn tweaked, thanks!

**Comment by brettkoonce** [comment]
@frankchn rebased to match master/HEAD again.  can merge this into #17815 if that would make things simpler.  I would normally rather do a lot of little commits (one for each contrib path) rather than touching a bunch of files like that PR is doing, do you have a preferred style?

**Comment by frankchn** [comment]
@brettkoonce Not really, we have a rotation of who handles PRs such as this, so everyone's preferences would be different. Just do whatever you prefer :)

**Comment by frankchn** [comment]
Unrelated test failure. Merging.

**Pull request merged**